modules/ROOT/examples/live-demos/annotations/index.html

```diff
@@ -8,7 +8,7 @@ <h5 style="text-align: center;">Note, this includes some "enterprise/premium" fe
   <h2>Got questions or need help?</h2>
   <ul>
     <li>Our <a href="//www.tiny.cloud/docs/">documentation</a> is a great resource for learning how to configure TinyMCE.</li>
```

Comment:
Update content links to reflect 6.0 urls
https://www.tiny.cloud/docs/tinymce/6/